scripts/build/termux_step_make.sh

```diff
@@ -8,8 +8,8 @@ termux_step_make() {
 
 	if test -f build.ninja; then
 		ninja -j $TERMUX_PKG_MAKE_PROCESSES
-	elif ls ./*.cabal &>/dev/null; then
-		cabal --config="$TERMUX_CABAL_CONFIG" build
+	elif ls ./*.cabal &>/dev/null || ls ./cabal.project &>/dev/null; then
```

Comment:
Note for later, we should absolutely not be using `ls` in any part of the build system.
At least we aren't attempting to parse it I guess.
###### See also: https://mywiki.wooledge.org/ParsingLs for further elaboration.